Bottom: 2e366a84216202cacbeca588034902a60caafb71
Top:    a9f20cc25349fe2594aeae1014c86512beaff711
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2014-04-09 16:01:25 -0700

Refresh of 0001-lazy-init.patch

---

diff --git a/src/cma.c b/src/cma.c
index e048879..b34472e 100644
--- a/src/cma.c
+++ b/src/cma.c
@@ -331,24 +331,6 @@ err1:
 	return ret;
 }
 
-//static struct ibv_context *ucma_get_verbs_by_index(int dev_index)
-//{
-//	ucma_dev_init(cma_dev_array[dev_index]);
-//	return cma_dev_array[dev_index].verbs;
-//}
-//
-//static struct ibv_context *ucma_get_verbs_by_guid(uint64_t guid)
-//{
-//	int i;
-//
-//	for (i = 0; i < cma_dev_cnt; i++) {
-//		if (cma_dev_array[i].guid == guid)
-//			return ucma_get_verbs_by_index(i);
-//	}
-//
-//	return NULL;
-//}
-
 struct ibv_context **rdma_get_devices(int *num_devices)
 {
 	struct ibv_context **devs;
@@ -369,7 +351,7 @@ struct ibv_context **rdma_get_devices(int *num_devices)
 	}
 	devs[i] = NULL;
 	if (num_devices)
-		*num_devices = devs ? cma_dev_cnt : 0;
+		*num_devices = cma_dev_cnt;
 	return devs;
 
 err2:
